### PR TITLE
Add missing Surface::get_device method

### LINKED ISSUE #312: How to get the device from a surface?
Hi,

for safety reasons, I wanted to `.finish()` the cairo XCB device, but I do not seem to be able to access it. For example, searching the code for `cairo_surface_get_device` does not find anything.

Are there no bindings for this function in cairo-rs?

Thanks & thanks for your work on cairo-rs.

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
Fixes #312 

cc @sdroege @EPashkin

### CODE REVIEW

**Comment by EPashkin** [comment]
@GuillaumeGomez Thanks, 👍

File: src/surface.rs [hunk 1]
```diff
@@ -150,6 +151,10 @@ impl Surface {
         }
     }
 
+    pub fn get_device(&self) -> Option<Device> {
+        unsafe { from_glib_none(ffi::cairo_surface_get_device(self.to_raw_none())) }
```
Comment (sdroege) [inline src/surface.rs:155]:
```suggestion
        unsafe {
            let device = ffi::cairo_surface_get_device(self.to_raw_none());
            if device.is_null() {
                None
            } else {
                Some(Device::from_raw_none(ffi::cairo_surface_get_device(self.to_raw_none())))
             }
        }
```

Comment (sdroege) [reply src/surface.rs:155]: Needs a `cargo fmt` run afterwards

**Comment by sdroege** [comment]
@GuillaumeGomez Good to go, only macos failed because of timeout as usual.

File: src/surface.rs [hunk 2]
```diff
@@ -150,6 +151,19 @@ impl Surface {
...
 
+    pub fn get_device(&self) -> Option<Device> {
+        unsafe {
+            let device = ffi::cairo_surface_get_device(self.to_raw_none());
+            if device.is_null() {
+                None
+            } else {
+                Some(Device::from_raw_none(ffi::cairo_surface_get_device(
+                    self.to_raw_none(),
+                )))
```
Comment (psychon) [inline src/surface.rs:162]: Uhm... How about `Some(Device::from_raw_none(device))` instead?

Comment (GuillaumeGomez) [reply src/surface.rs:162]: Outch indeed! I went way too quickly in here...

**Pull request merged**